R/utils.R

```diff
@@ -146,3 +146,10 @@ extract_baseline <- function(trial_file, baseline_file, quiet = TRUE) {
 catn <- function(...) {
   cat(..., "\n")
 }
+
+
+drop_path <- function(x) {
+  output <- sub("Path: [[:graph:]]*", "Path:", strsplit(x, "\n")[[1]]) |>
+    paste0(collapse = "\n")
```

Comment:
```suggestion
  output <- sub("Path: [[:graph:]]*", "Path:", x)
```
Seems to me that `x` is a character vector of lines, so we don't need to worry about `\n` splitting/pasting.
@darkgoldenrod-cherry